strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -48,12 +48,10 @@ class Circuit:
          using the Fock representation with given cutoff_dim.
          The state of the modes is manipulated by calling the various methods."""
     # pylint: disable=too-many-instance-attributes,too-many-public-methods
-    def __init__(self, graph, num_modes, cutoff_dim, pure=True, batch_size=None):
+    def __init__(self, graph, num_modes, cutoff_dim, pure, batch_size):
         self._graph = None # will be set when reset is called below, but reset needs something to compare to
-        self._batch_size = batch_size
-        self._batched = False if batch_size is None else True
         self._hbar = 2
-        self.reset(pure, graph, num_subsystems=num_modes, cutoff_dim=cutoff_dim)
+        self.reset(graph=graph, num_subsystems=num_modes, pure=pure, cutoff_dim=cutoff_dim, batch_size=batch_size)
```

Comment:
This was a tangentially related issue, now `TFBackend.reset` can also set/change the batch size.

Reply:
Issue #84